tests/test_fhir.py

```diff
@@ -0,0 +1,13 @@
+from sdc_services.models.r2.questionnaire_response import QuestionnaireResponse
+
+
+def test_r2_bundle(r2_questionnaire_response):
+    qnr = QuestionnaireResponse.from_json(r2_questionnaire_response)
+    assert len(qnr.group['question']) == 26
+
+def test_extract_api(client, r2_questionnaire_response):
+    result = client.post('/v/r2/fhir/$extract', json=r2_questionnaire_response)
+    assert result.status_code == 200
+
+    obs_bundle = result.json
+    assert len(obs_bundle['entry']) == 26
```

Comment:
👏 thank you for adding tests!!

The granularity of using the entire epic26 response makes it cumbersome to look for distinct values.  I'd suggest adding fixtures with just a single or small handful of responses (i.e. a hand made test only QNR) to test for expected values - beyond just a number of observations.

Reply:
Sure thing! Since this API is difficult to test by hand (POST'd JSON) I wanted to focus on testing.

I added another minimal QuestionnaireResponse and test that tests the generated Observation more deeply against the QuestionnaireResponse.